pr_agent/algo/ai_handler.py

```diff
@@ -99,28 +109,39 @@ async def chat_completion(self, model: str, system: str, user: str, temperature:
                 )
```

Comment:
**Suggestion:** In the `chat_completion` method, the `force_timeout` parameter is set to `get_settings().config.ai_timeout`. This means that the timeout value is constant for all the API calls. It might be beneficial to implement a dynamic timeout strategy, where the timeout value increases with each retry. This can be achieved by using an exponential backoff strategy for the timeout value. [enhancement]